### ISSUE TITLE
fatal: unable to create index.lock Another git process seems to be running in this repository

### ISSUE THREAD

**Issue by benmoot** [akonwi/git-plus#667]
I've tried multiple times to delete the index.lock using terminal and finder and still encounter this issue when trying to push commits to my repository. I have no clue where to go from here. I don't know what other processes could be running git. XCODE is closed, iTerm is closed. the only thing that would be using git is Atom

also worth noting that the file is changed and pushed from terminal to make sure everything is setup how it should be, but atom doesn't update to reflect the push, which is when I came across this error trying to keep Atom up-to-date with the repository

Steps to reproduce:

1. make change
2. try to push
3. fail.

![screen shot 2017-05-19 at 2 39 46 pm](https://cloud.githubusercontent.com/assets/8621691/26264437/72ec32ec-3ca2-11e7-8770-44a3c7fb2ca2.png)

**Comment by akonwi** [comment]
Have you tried to restart atom?

There are other issues that have been opened about this. In #213, multiple people found re-installing the package and restarting atom as a solution. 

It's hard to say what the cause is here. This package shouldn't be responsible for managing the index.lock file because it simply spawns a child process to execute the git commands.

**Comment by benmoot** [comment]
I was able to solve the problem and honestly it was 100% my fault. there were so many weird things that were going on with my .git files since I'm new to the whole process, that my solution for it was to delete the folders after pushing them all with terminal, then cloning the repos into my folders. after I did that and (after a lot of head pounding and trial and error), get my SSH key to finally assign itself, I was able to use the package as intended.
